examples/resources/aiven_kafka_quota/import.sh

```diff
@@ -0,0 +1 @@
+terraform import aiven_kafka_quota.example_quota PROJECT/SERVICE_NAME
```

Comment:
It should have client or user, or both. I would put all examples here.

Reply:
upd